infra/modules/feature-flags/access-policy.tf

```diff
@@ -0,0 +1,21 @@
+resource "aws_iam_policy" "access_policy" {
+  name        = "${local.evidently_project_name}-access"
+  path        = "/"
+  description = "Allows calls to EvaluateFeature on the Evidently project ${local.evidently_project_name}"
+
+  policy = data.aws_iam_policy_document.access_policy.json
+}
+
+#tfsec:ignore:aws-iam-no-policy-wildcards
+data "aws_iam_policy_document" "access_policy" {
+  statement {
+    sid    = "AllowEvaluateFeature"
+    effect = "Allow"
+    actions = [
+      "evidently:EvaluateFeature",
+    ]
+    resources = [
+      "${aws_evidently_project.feature_flags.arn}/feature/*",
```

Comment:
Do we want to lock this down to the variable list so we don't need a tfsec ignore?

Reply:
I thought about it, but I think it hurts readability for no security benefit.